views.py

```diff
@@ -0,0 +1,169 @@
+from flask import render_template, session, redirect, request
+from datetime import datetime
+
+from init import db, app
+from forms import LoginForm, RegistrationForm, OrderForm
+from tables import User, Dish, Category, Order
+from words import correct_word
+
+
+@app.route("/")
+def render_main():
+    items = db.session.query(Dish)
+    categories = db.session.query(Category)
+    session["word"] = correct_word(len(session.get("cart", [])))
```

Comment:
This is definitely not a backend task... This should be done inside templates

Reply:
Why not?
I didn't even think about it

Reply:
Cause it's pure representation-only issue. Don't make backend think about views and representations